src/test/resources/outputs/trino/basics/aggregation.sql

```diff
@@ -1,5 +1,7 @@
 --#[rel-aggregation-count-00]
-SELECT COUNT(*) AS "_1" FROM "default"."T" AS "T";
+SELECT count(1) AS "_1" FROM "default"."T" AS "T";
+-- previously was using COUNT(*)
+-- SELECT COUNT(*) AS "_1" FROM "default"."T" AS "T";
```

Comment:
self-review: similar to the Redshift PR (https://github.com/partiql/partiql-scribe/pull/90). changed tests are
- same as the PartiQL -> PartiQL translation PR
- comment out the exclude tests as these will be implemented in an upcoming PR
- remove redundant casts for BETWEEN since Trino supports `BETWEEN` without explicit casts on mixed numeric args